framework/auth/core.py

```diff
@@ -1331,6 +1336,29 @@ def n_projects_in_common(self, other_user):
         """Returns number of "shared projects" (projects that both users are contributors for)"""
         return len(self.get_projects_in_common(other_user, primary_keys=True))
 
+    def get_node_comment_timestamps(self, node, page):
```

Comment:
I think that should do it! I think I can also remove setting the timestamps for individual files here.